components/BUILD.yaml

```diff
@@ -95,6 +96,33 @@ packages:
         - ["sh", "-c", "echo \"version: ${version}\" >> versions.yaml"]
         - ["sh", "-c", "dev-version-manifest--app/version-manifest >> versions.yaml"]
         - ["sh", "-c", "rm -r components* dev-*"]
+  - name: needs-vuln-scan
```

Comment:
Uh, that's neat! But how do we make sure this stays complete? :bubbles: 

Maybe not a problem for now, but rather for Flex :+1: